### PR TITLE
Issues worklogs

### PR DESCRIPTION
**Pull request by killua99**
Implementation methods for worklogs.

### CODE REVIEW

**Comment by rjvandoesburg** [comment]
So is this still WIP or are you ready to submit?

**Comment by killua99** [comment]
Yeah, I mean, I get confuse, if you check the @see of the DocBlock I point to the API V3, but we should use the API v2, right, altho, the change between V2 to V3 is nothing for worklog

**Comment by rjvandoesburg** [comment]
Yeah, when I started v3 wasn't out yet, but I use v3 for every change now and all the old code still works, so you can just implement v3 for new code. 

That's also why I changed tagging from 1.x to 3.x to have the same number as Atlassian.

**Comment by killua99** [comment]
Alright, so do the CR check if the methods doesn't introduce any bug (I've been using them and all works just fine) and I guess ... it should be OK

File: src/Requests/Issue/IssueRequest.php [hunk 1]
```diff
@@ -191,4 +191,117 @@ public function notify($issueIdOrKey)
...
+    }
+
+    /**
+     * Returns all worklogs for an issue.
+     *
+     * @see https://developer.atlassian.com/cloud/jira/platform/rest/v3#api-rest-api-3-issue-issueIdOrKey-worklog-get
+     *
+     * @param string|int $issueIdOrKey
+     * @param int $workdlogId
+     * @param \Atlassian\JiraRest\Requests\Issue\Parameters\GetParameters|array $parameters
```
Comment (rjvandoesburg) [inline src/Requests/Issue/IssueRequest.php]: The documentation does not line up with the validation https://github.com/rjvandoesburg/laravel-jira-rest-client/pull/11/files#diff-fd7619950db52bec2613184c9a1032c4R235

This is the same for all methods

File: src/Requests/Issue/IssueRequest.php [hunk 2]
```diff
@@ -191,4 +191,117 @@ public function notify($issueIdOrKey)
...
+    {
+        $this->validateParameters($parameters, SearchParameters::class);
+
+        return $this->execute('post', "issue/{$issueIdOrKey}/worklog", $parameters);
+    }
+
+    /**
+     * Updates a worklog.
+     *
+     * @see https://developer.atlassian.com/cloud/jira/platform/rest/v3#api-rest-api-3-issue-issueIdOrKey-worklog-post
```
Comment (rjvandoesburg) [inline src/Requests/Issue/IssueRequest.php]: Incorrect url, use: https://developer.atlassian.com/cloud/jira/platform/rest/v3#api-rest-api-3-issue-issueIdOrKey-worklog-id-put

File: src/Requests/Issue/IssueRequest.php (hunk 1 above)
Comment (killua99) [reply src/Requests/Issue/IssueRequest.php]: You mean should it be the other way around? Ok what about now

File: src/Requests/Issue/IssueRequest.php (hunk 2 above)
Comment (killua99) [reply src/Requests/Issue/IssueRequest.php]: Fixed :)

**Comment by rjvandoesburg** [comment]
Created a new tag `3.0.0-beta4` so you can start using it right away!

**Pull request merged**